build.sh

```diff
@@ -138,6 +131,7 @@ set -x
 if [[ $BUILD_JCSDA == 'YES' ]]; then
   make -j ${BUILD_JOBS:-6} VERBOSE=$BUILD_VERBOSE
 else
+  builddirs="fv3-jedi soca iodaconv land-imsproc land-jediincr gdas-utils"
```

Comment:
do we even need fv3-jedi anymore? what about mpas? and why gdas-utils?

Reply:
I think we could still keep fv3-jedi as mentioned in the previous comments.   gdas-utils couldn't be used by rdas at all? 
Currently, as you pointed out previously, this option branch wouldn't not be used.